exspy/components/eels_cl_edge.py

```diff
@@ -167,15 +168,21 @@ def __init__(self, element_subshell, GOS="gosh", gos_file_path=None):
         self.fine_structure_coeff.ext_force_positive = False
         self.GOS = None
 
-        if GOS == "gosh":
-            self.GOS = GoshGOS(element_subshell, gos_file_path=gos_file_path)
+        if GOS == "dft":
```

Comment:
```suggestion
        if GOS == "gosh":
              warnings.warn(
                "The value 'gosh' of the `GOS` parameter has been renamed to 'dft' in "
                "eXSpy 0.3.0, use `GOS='dft'` instead. "
                "Using `GOS='gosh'` will stop working in eXSpy 1.0.",
                VisibleDeprecationWarning,
            )
            GOS = "dft"
        if GOS == "dft":
```

Corresponding import needs to be added at the beginning of the file:
```python
from hyperspy.exceptions import VisibleDeprecationWarning
```